core/src/main/java/org/apache/accumulo/core/client/BatchWriterConfig.java

```diff
@@ -209,15 +231,15 @@ public void write(DataOutput out) throws IOException {
     // write this out in a human-readable way
     ArrayList<String> fields = new ArrayList<>();
     if (maxMemory != null)
-      addField(fields, "maxMemory", maxMemory);
+      addField(fields, MAX_MEM.getKey(), maxMemory);
```

Comment:
Could leave this method unchanged.  Not sure if there is a benefit to changing these keys

Reply:
Yeah its not work breaking serialized data.  I will undo the changes to the serialized code.